crates/swift-bridge-ir/src/codegen/generate_rust_tokens.rs

```diff
@@ -93,6 +93,27 @@ impl ToTokens for SwiftBridgeModule {
 
                     match ty.host_lang {
                         HostLang::Rust => {
+                            if ty.attributes.hashable {
+                                let export_name = format!("__swift_bridge__${}$_hash", ty_name);
+                                let function_name = syn::Ident::new(
+                                    &format!("__swift_bridge__{}__hash", ty_name),
+                                    ty.ty.span(),
+                                );
+                                let tokens = quote! {
+                                use std::hash::Hash;
```

Comment:
You can move `use std::hash::{Hash, Hasher}` inside of the generated function so that we don't have multiple includes in the same scope when there are multiple Hashable attributes.

Reply:
> You can move use `std::hash::{Hash, Hasher}` inside of the generated function

Do you mean this?:
```
pub extern "C" fn #function_name (
    this: *const super::#ty_name,
) -> u64 {
    use std::hash::{Hash, Hasher};
    let mut s = std::collections::hash_map::DefaultHasher::new();
    (unsafe {&*this}).hash(&mut s);
    s.finish()
}
```

Reply:
Yup

Reply:
You can also make it `use std::collections::hash_map::DefaultHasher` if you want.

I'm fine either way as long we we keep the use statements inside of the function so that we don't the same use multiple times in the module's root scope.